src/main/java/org/matsim/prepare/CreateSupply.java

```diff
@@ -47,6 +49,7 @@
 
 	private static final Path osmData = Paths.get("public-svn/matsim/scenarios/countries/de/metropole-ruhr/metropole-ruhr-v1.0/original-data/osm/nordrhein-westfalen-2021-02-15.osm.pbf");
```

Comment:
adding "public-svn" or "shared-svn" to the path assumes that everybody has checked out all our repos in a single folder. I guess most people do, but one could also go without and requrie both root folders as input. 

A matter of taste I guess